### PR TITLE
doc: wifi: Update Certification page

### PR DESCRIPTION
**Pull request by richabp**
Updated Certification page with information about
recertification process and related links.

SHEL-2677

### CODE REVIEW

File: doc/nrf/protocols/wifi/wifi_certification.rst [hunk 1]
```diff
@@ -11,20 +11,32 @@ The Wi-Fi Alliance® offers a full `certification program <Wi-Fi Certification_>
...
 Derivative certification is applicable for end products based on third-party Wi-Fi CERTIFIED™ modules containing an nRF70 Series IC without any modification.
 
 For details about the Wi-Fi certification program and the available paths for the nRF70 Series, read the `Wi-Fi Alliance Certification for nRF70 Series`_ document.
 
 For instructions on how to specifically use the QuickTrack certification path, see the :ref:`wifi_wfa_qt_app_sample` page.
+
+Recertification process after end product software upgrade
+**********************************************************
+
+When a customer’s Wi-Fi CERTIFIED end product requires an update with new software corresponding to a recent |NCS| release, the customer must initiate the Wi-Fi CERTIFIED end product variant process.
```
Comment (ioannisg) [inline doc/nrf/protocols/wifi/wifi_certification.rst:35]:
```suggestion
When a customer’s Wi-Fi CERTIFIED end product receives an update with new software corresponding to a recent |NCS| release, the customer must initiate the Wi-Fi CERTIFIED end product variant process.
```

Comment (ioannisg) [reply doc/nrf/protocols/wifi/wifi_certification.rst:35]: I think of "receives" here, to point out better the temporal relation, i.e. first a product receives an update, then a re-certification process must kick off.

File: doc/nrf/protocols/wifi/wifi_certification.rst [hunk 2]
```diff
@@ -11,20 +11,32 @@ The Wi-Fi Alliance® offers a full `certification program <Wi-Fi Certification_>
...
+This recertification process involves rerunning the QuickTrack Wi-Fi certification testing on the end product with the updated software and basing it on the Qualified Solution or variant from Nordic Semiconductor corresponding to that version of the |NCS| software.
```
Comment (ioannisg) [inline doc/nrf/protocols/wifi/wifi_certification.rst:36]: @bama-nordic what is the process when the end product is based on a derivative certification path?

Comment (ioannisg) [inline doc/nrf/protocols/wifi/wifi_certification.rst:36]:
```suggestion
Under the QuickTrack certification path,  this recertification process involves rerunning the QuickTrack Wi-Fi certification testing on the end product with the updated software and basing it on the Qualified Solution or variant from Nordic Semiconductor corresponding to that version of the |NCS| software.
```

File: doc/nrf/protocols/wifi/wifi_certification.rst [hunk 3]
```diff
@@ -11,20 +11,32 @@ The Wi-Fi Alliance® offers a full `certification program <Wi-Fi Certification_>
...
+
+The certification process is controlled by the Wi-Fi Alliance.
```
Comment (ioannisg) [inline doc/nrf/protocols/wifi/wifi_certification.rst:38]: I would actually move this sentence before the above (long) one that details the process.

**Review by ioannisg — COMMENTED** [review]
Thanks. Added a couple of comments.

File: doc/nrf/protocols/wifi/wifi_certification.rst (hunk 2 above)
Comment (bama-nordic) [reply doc/nrf/protocols/wifi/wifi_certification.rst]: @ioannisg , From my understanding, the customer needs to just re-apply for derivative certification basing it on the new qualified variant and does not need to run QT testing. This is not very clear from WFA documentation and I am writing to them to clarify. The below link regarding derivative certification process is good to include here -
https://www.wi-fi.org/system/files/Wi-Fi_CERTIFIED_Derivative_Certifications_Overview_v3.3.pdf

**Review by ioannisg — APPROVED** [review]

**Comment by ioannisg** [comment]
@bama-nordic I approved (so we can get the PR merged), with the condition that we can revisit this later with respect to Derivative path.

**Review by bama-nordic — APPROVED** [review]

**Review by kittydepa — APPROVED** [review]

**Pull request merged**